### ISSUE TITLE
Documentation Enhancement: Correct YAML Indentation in queue.yaml Example

### ISSUE THREAD

**Issue by rwexler** [pyiron/pyiron_base#1114]
Hello Pyiron team,

First, I want to express my appreciation for the robust framework that you have created, and I am excited to further explore its capabilities.

However, I've encountered an issue regarding the queue.yaml configuration, specifically on the YAML indentation in the installation and testing instructions, found here: https://pyiron.readthedocs.io/en/latest/source/installation.html.

The current example in the documentation reads:

```
queue_type: SLURM
queue_primary: work
queues:
work: {cores_max: 128, cores_min: 1, run_time_max: 1440, script: work.sh}
express: {cores_max: 128, cores_min: 1, run_time_max: 1440, script: express.sh}
```

As a YAML novice, I initially didn't realize that work and express should be indented under queues. When they are not indented, they are interpreted as dictionary keys at the same level as queues, which results in the value of queues being None. This caused some confusion and a longer troubleshooting process than necessary.

I suggest updating the documentation to illustrate the correct indentation, which should be:

```
queue_type: SLURM
queue_primary: work
queues:
  work: {cores_max: 128, cores_min: 1, run_time_max: 1440, script: work.sh}
  express: {cores_max: 128, cores_min: 1, run_time_max: 1440, script: express.sh}
```

I believe this small change will significantly improve the onboarding experience for new users who might not be familiar with the intricacies of YAML.

Thank you for considering this suggestion, and I look forward to your response.

Best Regards,
Rob

**Comment by niklassiemer** [comment]
Hello @rwexler !

Welcome to pyiron and thanks for the feedback! I will hava a look on Monday and fix the documentation flaw.

Best
Niklas

**Comment by jan-janssen** [comment]
@niklassiemer When you update the queuing system reference can you also add a link to https://pysqa.readthedocs.io/en/latest/queue.html

**Comment by niklassiemer** [comment]
https://github.com/pyiron/pyiron/pull/1495 should fix this.

**Comment by ligerzero-ai** [comment]
Thanks for your interest and keen eye! Sorry about the inconvenience caused by the oversight. Let us know if you run into any more issues, happy to help! Solved by pyiron/pyiron#1498

**Comment by rwexler** [comment]
Thanks, everyone!
